plugins/localselectors/README.md

```diff
@@ -0,0 +1,37 @@
+# Sample Plugin
+
+This plugin modifies `Workplane` selectors so that they can be used to specify axes in the local coordinate place.
```

Comment:
```suggestion
This plugin modifies `Workplane` selectors so that they can be used to specify axes in the local coordinate plane.
```

Makes more sense in my head, not sure though, as English isn't my native language, so maybe I'm missing something.

Reply:
Good catch